src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -257,6 +258,8 @@ def load_arguments(self, _):
         c.argument('ppg', type=str, validator=validate_ppg)
         c.argument('vnet_subnet_id', type=str,
                    validator=validate_vnet_subnet_id)
+        c.argument('pod_subnet_id', type=str,
+                   validator=validate_pod_subnet_id)
```

Comment:
Because `type=str` is the default value, so it does not need to be defined repeatedly

Reply:
On the line above we have:
```
        c.argument('vnet_subnet_id', type=str,
                   validator=validate_vnet_subnet_id)
```
Shouldn't we just keep them consistent? Or would you like me to update both?

Reply:
If it's convenient for you, please update both~ If not, we can modify these in another new separate PR

Reply:
Sounds good I we can do cleanup in a separate PR.